apps/finance/app/src/components/ToggleFiltersButton.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import styled from 'styled-components'
+import { theme } from '@aragon/ui'
+
+const StyledButton = styled.button`
+  border: none;
+  background: none;
+  height: 16px;
+  width: 14px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 0;
+  cursor: pointer;
+  outline: none;
+
+  &:focus {
+    border: 2px solid ${theme.accent};
+  }
+  &:active {
+    border: none;
+  }
+`
+
+export default props => (
+  <StyledButton {...props}>
+    <svg
```

Comment:
Just as a note, we should move this icon in Aragon UI (but we can keep it like this until it gets released).